### DISCUSSION TITLE
Link Prompt & Trace with langchain CallbackHandler

### DISCUSSION THREAD

**Discussion by derevyan** [langfuse/langfuse#2354] [Support] [answered]
I found that prompt tracing is a great feature, but try to find a way to create a link to prompt tracing data using - CallbackHandler for langchain, any way to make it? 🙏

**Comment by marcklingen** [comment]
thanks for raising this, let's merge the conversation into this thread: https://github.com/orgs/langfuse/discussions/2237

**Comment by marcklingen** [comment, marked as answer]
This was just released: https://github.com/orgs/langfuse/discussions/2237#discussioncomment-10668561
